Tests/src/PublicAPITests/QuotaSystemTests.cpp

```diff
@@ -42,10 +42,10 @@ bool RequestPredicate(const csp::services::ResultBase& Result)
 #if RUN_ALL_UNIT_TESTS || RUN_QUOTASYSTEM_TESTS || RUN_QUOTASYSTEM_QUERY_TEST
 CSP_PUBLIC_TEST(CSPEngine, QuotaSystemTests, TierNameEnumTesttoStringTest)
 {
-	EXPECT_EQ(TierNameEnumToString(csp::systems::TierNames::Basic), "Basic");
-	EXPECT_EQ(TierNameEnumToString(csp::systems::TierNames::Premium), "Premium");
-	EXPECT_EQ(TierNameEnumToString(csp::systems::TierNames::Pro), "Pro");
-	EXPECT_EQ(TierNameEnumToString(csp::systems::TierNames::Enterprise), "Enterprise");
+	EXPECT_EQ(TierNameEnumToString(csp::systems::TierNames::Basic), "basic");
```

Comment:
Should we talk to CHS about making these case-insensitive?

Reply:
Yes, please do!

Reply:
Added a topic to the engineering sync for this